Source/WebGPU/WebGPU/WebGPU.h

```diff
@@ -1866,8 +1866,27 @@ WGPU_EXPORT WGPUBool wgpuXRViewIsValid(WGPUXRView view) WGPU_FUNCTION_ATTRIBUTE;
 #endif  // !defined(WGPU_SKIP_DECLARATIONS)
 #ifdef __cplusplus
 } // extern "C"
+#endif
+
+#ifdef __cplusplus
 WGPU_EXPORT std::span<uint8_t> wgpuBufferGetBufferContents(WGPUBuffer buffer) WGPU_FUNCTION_ATTRIBUTE;
 WGPU_EXPORT std::span<uint8_t> wgpuBufferGetMappedRange(WGPUBuffer buffer, size_t offset, size_t size) WGPU_FUNCTION_ATTRIBUTE;
+
+inline auto span(const WGPUDeviceDescriptor& deviceDescriptor) { return unsafeForgeSpan(deviceDescriptor.requiredFeatures, deviceDescriptor.requiredFeatureCount); }
+inline auto span(const WGPUBindGroupDescriptor& bindGroupDescriptor) { return unsafeForgeSpan(bindGroupDescriptor.entries, bindGroupDescriptor.entryCount); }
+inline auto span(const WGPUBindGroupLayoutDescriptor& bindGroupLayoutDescriptor) { return unsafeForgeSpan(bindGroupLayoutDescriptor.entries, bindGroupLayoutDescriptor.entryCount); }
+inline auto span(const WGPURenderPassDescriptor& renderPassDescriptor) { return unsafeForgeSpan(renderPassDescriptor.colorAttachments, renderPassDescriptor.colorAttachmentCount); }
+inline auto span(const WGPURenderBundleEncoderDescriptor& renderBundleEncoderDescriptor) { return unsafeForgeSpan(renderBundleEncoderDescriptor.colorFormats, renderBundleEncoderDescriptor.colorFormatCount); }
```

Comment:
Would propose naming this `colorFormatsSpan` since it is not obvious from the name what container is being returned as a span.

Similar suggestions apply for the other `span` functions where it is not obvious from the function parameter which container is being used to create the span